docs-src/tctl-v1/workflow/signal.md

```diff
@@ -10,6 +10,8 @@ tags:
 The `tctl workflow signal` command [Signals](/concepts/what-is-a-signal) a [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
 Workflows listen for Signals by their Signal name, and can be made to listen to one or more Signal names.
+Workflows can also listen for Queries.
```

Comment:
Is this a [Query](https://docs.temporal.io/workflows#query) or just a --query (a SQL query) that can be passed with the Signal?